code/week5/week5.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
If an additional file (`main2.cpp`) is added which includes `week5.h` and does not `#pragma once` in it's own cpp file, it won't get the `#pragma once`, which can be a gotchya.